frontend/src/components/simple-auto-filter/simple-auto-filter.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { FC, useEffect, useMemo, useState } from 'react';
+
+import { AutoRiaOption } from '@autoline/shared/common/types/types';
+import { FiltersNames } from '@common/enums/cars/filters-names.enum';
+import { pricesRange, yearsRange } from '@common/enums/cars/ranges';
+import { AutocompleteValueType } from '@common/types/cars/autocomplete.type';
+import { RangeValueType } from '@common/types/cars/range-item.type';
+import { AutocompleteInput } from '@components/common/autocomplete-input/autocomplete-input';
+import { RangeSelector } from '@components/common/range-selector/range-selector';
+import { SelectField } from '@components/common/select-field/select-field';
+import { filtersToQuery } from '@helpers/filters-to-query';
+import { getValueById } from '@helpers/get-value-by-id';
+import { useAppDispatch, useAppSelector } from '@hooks/store/store.hooks';
+import { setValue } from '@store/car-filter/slice';
+import {
+  useGetBrandsQuery,
+  useGetFilteredCarsQuery,
+  useGetModelsOfBrandQuery,
+  useGetUsedOptionsQuery,
+} from '@store/queries/cars';
+
+import styles from './styles.module.scss';
+
+const SimpleAutoFilter: FC = () => {
+  const dispatch = useAppDispatch();
+
+  const filters = useAppSelector((state) => state.carFilter);
+
+  const [queryParams, setQueryParams] = useState<string[][]>([]);
+
+  const { data: brands, isLoading } = useGetBrandsQuery();
+  const { data: options, isLoading: isOptionsLoading } =
+    useGetUsedOptionsQuery();
+
+  const { data: models } = useGetModelsOfBrandQuery(filters.brandId[0], {
+    skip: !filters.brandId.length,
+  });
+
+  useEffect(() => {
+    setQueryParams(filtersToQuery(filters));
+  }, [filters]);
+
+  const { data: filteredCars } = useGetFilteredCarsQuery(queryParams, {
+    skip: !queryParams,
+  });
+
+  // eslint-disable-next-line no-console
+  console.log(filteredCars);
+
+  const years = useMemo(() => yearsRange(30), []);
+
+  const selectedRegion = useMemo(() => {
+    if (!isLoading && options) {
+      const region = options?.regions?.find(
+        (region) => region.id === filters.regionId,
+      );
+      if (region) {
+        return {
+          label: region.name,
+          id: region.id,
+        };
+      }
+    }
+    return {
+      label: '',
+      id: '',
+    };
```

Comment:
Please use guard clauses instead of having redundant if nesting -https://learningactors.com/javascript-guard-clauses-how-you-can-refactor-conditional-logic/

Reply:
Yes, I'll change it. Unfortunately, I lost a few code-style changes due to conflicts in old **PR**, and these lines were one of them.